tests/rules/no-missing-link-fragments.test.js

```diff
@@ -0,0 +1,366 @@
+/**
+ * @fileoverview Tests for no-missing-link-fragments rule.
+ * @author Sweta Tanwar (@SwetaTanwar)
+ */
+
+//------------------------------------------------------------------------------
+// Imports
+//------------------------------------------------------------------------------
+
+import rule from "../../src/rules/no-missing-link-fragments.js";
+import markdown from "../../src/index.js";
+import { RuleTester } from "eslint";
+import dedent from "dedent";
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+	plugins: {
+		markdown,
+	},
+	language: "markdown/commonmark",
+});
+
+ruleTester.run("no-missing-link-fragments", rule, {
+	valid: [
+		// Basic heading match
+		dedent`
+            # Heading Name
+            [Link](#heading-name)
+        `,
```

Comment:
```suggestion
		dedent`
                # Heading Name
                [Link](#heading-name)`,
```

Could you align the strings using `dedent`?

It would be great if you could follow the same alignment style you used in [`no-missing-atx-heading-space.test.js`](https://github.com/eslint/markdown/blob/main/tests/rules/no-missing-atx-heading-space.test.js).